### PR TITLE
feat(utils): Move common classes and functions to utils

### PR DESCRIPTION
**Pull request by danadi7**
#### What this PR does / why we need it:
- Create an Ordexer Utils package for common classes and functions
- Move the Promise Limiter from Indexer to Ordexer Utils for use in API
- Move BTC-Sat conversion functions to Bitcoin Service utils
- Add Rarity utils to Ord Service

#### Which issue(s) does this PR fixes?:

Fixes ORD-

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] No console errors on web
- [ ] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*

### CODE REVIEW

File: packages/ordexer-utils/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      "prettier --write --ignore-unknown"
+    ],
+    "*.{js,jsx,ts,tsx}": [
+      "eslint --fix"
+    ]
+  },
+  "prettier": "@ordzaar/standard-prettier",
+  "dependencies": {
+    "@ordzaar/standard-api-fastify": "^0.4.11",
+    "axios": "^1.6.5",
```
Comment (joundy) [inline packages/ordexer-utils/package.json:26]: these lib dependencies are unused

Comment (thedoublejay) [reply packages/ordexer-utils/package.json:26]: we use fastify

Comment (thedoublejay) [reply packages/ordexer-utils/package.json:26]: it's the base buindle

Comment (joundy) [reply packages/ordexer-utils/package.json:26]: yap, but this is shared library for utils not for the nest package, i think it's different

File: apps/api/package.json [hunk 2]
```diff
@@ -21,17 +21,18 @@
   "prettier": "@ordzaar/standard-prettier",
   "dependencies": {
     "@defichain/jellyfish-json": "^4.0.5",
-    "@fastify/multipart": "^8.0.0",
-    "@logtail/pino": "^0.4.17",
+    "@fastify/multipart": "^8.1.0",
```
Comment (thedoublejay) [inline apps/api/package.json:24]: can remove this also, it's added on standard fastify now

File: packages/ord-service/src/providers/OrdProvider.ts [hunk 3]
```diff
@@ -126,3 +127,20 @@ export type OrdInscription = {
   satpoint: string;
   timestamp: number;
 };
+
+export type Ordinal = {
```
Comment (thedoublejay) [inline packages/ord-service/src/providers/OrdProvider.ts:131]: Actually, this is already in Ordit SDK. Do you think we can just use that?

Comment (thedoublejay) [reply packages/ord-service/src/providers/OrdProvider.ts:131]: https://github.com/ordzaar/ordit-sdk/pull/55

Comment (joundy) [reply packages/ord-service/src/providers/OrdProvider.ts:131]: yapp, but i think it's better to update in different PR, need to change some type in the indexer/api too

File: apps/api/package.json (hunk 2 above)
Comment (joundy) [reply apps/api/package.json]: updated

File: packages/ord-service/src/providers/OrdProvider.ts (hunk 3 above)
Comment (thedoublejay) [reply packages/ord-service/src/providers/OrdProvider.ts:131]: ok ok

**Review by joundy — APPROVED** [review]

**Pull request merged**